data/tag_windows.txt

```diff
@@ -1,12 +1,21 @@
 # Tags Windows application execution events.
+# trace of execution & task schedule
+# Can help you to identify suspect command runned (wscript, hh, powershell, cmd, ...)
+# Knowledge: https://jpcertcc.github.io/ToolAnalysisResultSheet/
 application_execution
   data_type is 'fs:stat' AND filename contains PATH('Windows/Tasks/At')
-  data_type is 'windows:evt:record' AND source_name is 'Security' AND event_identifier is 592
-  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Security-Auditing' AND event_identifier is 4688
+  (data_type is 'windows:evt:record' OR data_type is 'windows:evtx:record') AND source_name is 'Security' AND event_identifier is 592
```

Comment:
Why this change? Isn't this ID only used by Windows versions that use Evt?

https://www.ultimatewindowssecurity.com/securitylog/encyclopedia/event.aspx?eventid=592

Reply:
Hi, sorry for event 592, you are right, it doesnt need to:
```OR data_type is 'windows:evtx:record')```

For event "4688", I put on one line with other event:
```
data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Security-Auditing' AND (event_identifier is 4673 OR event_identifier is 4688)
```